docs/kubernetes/user-guides/basic.md

```diff
@@ -85,3 +85,9 @@ $ kubectl get pods
 NAME                      READY     STATUS    RESTARTS   AGE
 web-server                1/1       Running   0          1m
 ```
+
+## StorageClass Fields
+
+The list of recognized StorageClass [`parameters`](https://kubernetes.io/docs/concepts/storage/storage-classes/#parameters) is the same as the list of [CSI CreateVolume parameters](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver#createvolume-parameters).
```

Comment:
You can probably use relative path here. Then it will display the correct page on a specific tag.